_i18n/pt_BR/factors/backwards-compatible-apis.md

```diff
@@ -0,0 +1,23 @@
+Apesar da maioria dos seus usuários atualizarem para a última versão dentro de poucas semanas, sempre existirá um grupo de usuários que não o fará. Isso pode ter inúmeras razões. Algumas vezes isso está relacionado com a versão do iOS que eles usam, versão essa que eles nem sempre podem atualizar dependendo do quão antigo o aparelho é.
```

Comment:
I would change `sempre existirá um grupo de usuários que não o fará. Isso pode ter inúmeras razões.` to `sempre existirá um grupo de usuários que não o fará, por inúmeras razões.` I know it is a slight different of the english version, but at least for me it sounds a bit better.